pkg/scheduler/predicates/ha.go

```diff
@@ -180,7 +197,12 @@ func (h *ha) realAcquireLock(pod *apiv1.Pod) (*apiv1.PersistentVolumeClaim, *api
 	if schedulingPVC == currentPVC {
 		return schedulingPVC, currentPVC, nil
 	}
-	if schedulingPVC.Status.Phase != apiv1.ClaimBound {
+	schedulingPodName := getPodNameFromPVC(schedulingPVC)
+	schedulingPod, err := h.podGetFn(ns, schedulingPodName)
+	if err != nil {
+		return schedulingPVC, currentPVC, err
+	}
+	if schedulingPVC.Status.Phase != apiv1.ClaimBound || schedulingPod.Spec.NodeName == "" {
```

Comment:
Can you add some comments for function `realAcquireLock`?  I feel that the logic in this function is difficult to understand